sdk/registry.go

```diff
@@ -22,9 +22,7 @@ var moduleRe = regexp.MustCompile(`/modules/(([^/]+)/hooks/([^/]+/)*([^/]+))$`)
 // TODO: This regexp should be changed. We shouldn't force users to name modules with a number prefix.
 var moduleNameRe = regexp.MustCompile(`^[0-9][0-9][0-9]-(.*)$`)
 
-var _ = initRegistry()
-
-func initRegistry() bool {
+var _ = func() struct{} {
```

Comment:
This initialization function and stubs below are redundant now.

Reply:
Removed all useless stubs and Register(). Only RegisterFunc() is left now since there is no practical need for a complete object.